container-deployment/docker-compose.md

```diff
@@ -4,7 +4,10 @@ This article explains how to collect [Docker](https://www.docker.com/) logs and
 
 ![Kibana](../.gitbook/assets/7.10_kibana-homepage.png)
 
-[Elasticsearch](https://www.elastic.co/products/elasticsearch) is an open-source search engine known for its ease of use. [Kibana](https://www.elastic.co/products/kibana) is an open-source Web UI that makes Elasticsearch user-friendly for marketers, engineers and data scientists alike.
+[Elasticsearch](https://www.elastic.co/products/elasticsearch) was an open-source search engine known for its ease of use. [Kibana](https://www.elastic.co/products/kibana) was an open-source Web UI that makes Elasticsearch user-friendly for marketers, engineers and data scientists alike.
```

Comment:
was -> had been x2 

With small tense fixes, it should be fine.

Reply:
Thanks, I've fixed it.